plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
And what happens if the next version of the driver buggy again? would you use it again for all "ATI" drivers, assuming that everyone has the latest driver and therefore it must be buggy for everyone? And what if some people avoid the update specifically to stay with the non-buggy version?

Besides, the way you changed it leaves an unused variable which you initialize and assigns values to it etc, but never ending up using it.

It doesn't have to be a test suite. You set it according to the vendor, so you can probably also check for version range.